mon-pix/translations/en.json

```
@@ -32,313 +32,313 @@
 
   "pages": {
     "assessment-results": {
-      "title": "End of assessment",
+      "title": "End of test",
       "actions": {
         "continue-pix-experience": "Continue my Pix experience",
-        "return-to-homepage": "Return to homepage"
+        "return-to-homepage": "Return to home"
       },
       "answers": {
         "header": "Your answers"
       }
     },
 
     "campaign": {
-      "title": "Parcours",
+      "title": "Course",
       "errors": {
-        "not-accessible": "Oups, the requested page is not accessible.",
-        "no-longer-accessible": "Oups, the requested page is no longer accessible."
+        "not-accessible": "Oops, the requested page is not available.",
+        "no-longer-accessible": "Oops, the requested page is not available."
       }
     },
 
     "campaign-landing": {
       "title": "Presentation",
       "assessment": {
-        "title": "Start your journey",
-        "action": "I begin",
-        "announcement": "Start your personalized assessment path. <br /> Register or log into the Pix platform and start your test.",
-        "details": "During this course, you will have the opportunity to:",
-        "legal": "Information relating to your progress will be sent to the course organizer to enable him to accompany you. ",
+        "title": "Begin your course",
+        "action": "Begin",
+        "announcement": "Start your personalized assessment course.<br /> Sign up or log in to the Pix platform and start your test.",
+        "details": "During this course, you will be able to:",
+        "legal": "Information about your progress will be sent to the course administrator so they can provide you with support. Your test results will only be shared with your permission.",
```

Le `with` me semble étrange 🤔 
```suggestion
        "legal": "Information about your progress will be sent to the course administrator so they can provide you some support. Your test results will only be shared with your permission.",
```

---

Le `with` ne me choque pas